openfisca_web_api/handlers.py

```diff
@@ -6,13 +6,13 @@
 
 import dpath
 
-from openfisca_core.simulations import Simulation
+from openfisca_core.simulation_builder import SimulationBuilder
 from openfisca_core.indexed_enums import Enum
 from openfisca_core.commons import to_unicode
 
 
 def calculate(tax_benefit_system, input_data):
-    simulation = Simulation(tax_benefit_system = tax_benefit_system, simulation_json = input_data)
+    simulation = SimulationBuilder().build_from_entities(tax_benefit_system, input_data)
```

Comment:
Good catch, thanks! I'm puzzled, why wasn't that caught by the web API tests?

Reply:
Oh, never mind. We went from using the old API to using the SimulationBuilder, not from one form of SimulationBuilder to another. The tests will break in due time when we continue refactoring ;)